lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -71,7 +70,7 @@ func ImageCache() *ICache {
 
 // Update runs only once at startup to hydrate the image cache
 func (ic *ICache) Update(client *client.PortLayer) error {
-	log.Debugf("Updating image cache...")
+	log.Debugf("Updating image cache")
```

Comment:
Minor and feel free to ignore.  This could probably be made an `Infof` message.   Up to you.